sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt

```diff
@@ -10,24 +11,31 @@ internal actual object SentryBridge {
         return SentryId(androidSentryId.toString())
     }
 
+    actual fun captureMessage(message: String, scopeCallback: SentryScopeCallback): SentryId {
+        val localScope = SentryScope()
+        val scopeConfiguration = localScope.scopeConfiguration(scopeCallback)
```

Comment:
I'm not sure this would work, See Java SDK
https://github.com/getsentry/sentry-java/blob/fa3886ffaae668ef2e5d57783df89bfd8bb0c6bb/sentry/src/main/java/io/sentry/Hub.java#L820-L828
First, we have to `clone` the current scope and later apply the callback, is it working this way?

Reply:
Isn't this being done internally since we modify the scope directly? When I tested it the scope was independently isolated from others only for that event but I'll have to check again